src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertTo-Html.cs

```diff
@@ -449,6 +472,8 @@ protected override void BeginProcessing()
 
             _propertyMshParameterList = ProcessParameter(_property);
 
+            regHttp = new Regex(@"(?<=\()\b(https?://|www\.)[-A-Za-z0-9+&@#/%?=~_()|!:,.;]*[-A-Za-z0-9+&@#/%=~_()|](?=\))|(?<=(?<wrap>[=~|_#]))\b(https?://|www\.)&[-A-Za-z0-9+&@#/%?=~_()|!:,.;]*[-A-Za-z0-9+&@#/%=~_()|](?=\k<wrap>)|\b(https?://|www\.)[-A-Za-z0-9+&@#/%?=~_()|!:,.;]*[-A-Za-z0-9+&@#/%=~_()|]", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
Should we just use the one that Windows Terminal uses?  https://github.com/microsoft/terminal/blob/a01500f051bc896465ac4bd37811142bc2347fb8/src/cascadia/TerminalCore/Terminal.hpp#L22

Reply:
I think the windows terminal one also detects file? I think we only need URLs.

Reply:
@StevenBucher98 do you have any opinions on this?

Reply:
I dont see a glaring problem including the files like how Windows Terminal does, I sorta like the consistency with URLs and files being wrapped in hyperlinks. IMO

Reply:
I tried the Windows terminal regex but it does not support URLs with parenthesis, for example,  http://example.com/test(1).html.  It only recognizes the part that is before the parenthesis, thus the result would be <a href="http://example.com/test">http://example.com/test</a>(1).html

So I did some minor modifications to the windows terminal regex to add support for parenthesis in URLs

Reply:
@JamesWTruher and I already had a conversation about the regex, and we both agreed on this regex solution.